pymc/distributions/distribution.py

```diff
@@ -226,8 +226,8 @@ def __new__(
         transform : optional
             See ``Model.register_rv``.
         **kwargs
-            Keyword arguments that will be forwarded to ``.dist()``.
-            Most prominently: ``shape`` and ``size``
+            Keyword arguments that will be forwarded to ``.dist()`` or the Aesara RV Op.
+            Most prominently: ``shape`` for ``.dist()`` and ``size`` or ``dtype`` for the Op.
```

Comment:
This could be confusing, users cant specify both shape and size. Better to not mention at all?

Reply:
```suggestion
            Most prominently: ``shape`` for ``.dist()`` or ``dtype`` for the Op.
```